src/mono/sample/HelloWorld/HelloWorld.csproj

```diff
@@ -25,6 +25,8 @@
       LibraryFormat="$(_AotLibraryFormat)"
       Assemblies="@(AotInputAssemblies)"
       OutputDir="$(PublishDir)"
+      CollectCompiledMethods="$(StripILCode)"
+      CompiledMethodsOutputPath="$(MethodTokenFilePath)"
```

Comment:
Why the difference in naming - `MethodTokenFilePath` vs `CompiledMethodsOutputPath`?

Reply:
Fixed.